hikari/internal/mentions.py

```diff
@@ -69,20 +69,20 @@ def generate_allowed_mentions(
     parsed_mentions: typing.List[str] = []
     allowed_mentions: typing.Dict[str, typing.Any] = {"parse": parsed_mentions}
 
-    if mentions_everyone is True:
+    if mentions_everyone:
         parsed_mentions.append("everyone")
 
-    if mentions_reply is True:
+    if mentions_reply:
         allowed_mentions["replied_user"] = True
 
-    if user_mentions is True:
+    if user_mentions:
```

Comment:
`is True` is definitely needed here as it has significance when compared to the variable just being truthy (see the elif clause)

Reply:
- [x] Fixed